python/mooseutils/MooseDataFrame.py

```diff
@@ -0,0 +1,89 @@
+import os
+import pandas
+
+import tools
+
+class MooseDataFrame(object):
+    """
+    A wrapper for handling data from a single csv file.
+
+    This utilizes a pandas.DataFrame for storing and accessing CSV data, while
+    allowing for the file to exist/not-exist.
+    """
+
+    NOCHANGE = 0
+    INVALID = 1
+    OLDFILE = 2
+    UPDATED = 3
+
+    def __init__(self, filename, index=None, run_start_time=None):
+
+        self.filename = filename
+        self.modified = None
+        self.data = pandas.DataFrame()
+        self._index = index
+        self._run_start_time = run_start_time
```

Comment:
You have a test for this in `test_MooseDataFrame.py`. It doesn't actually seem to be used by anything though. Just get rid of the functionality entirely?